roguedex-dev/roguedex
# Update type icons to match Pokerogue sprites

### What changed
I've copied the type icons from Pokerogue and added them as resources in the Chrome extension, and we're loading those instead of hitting the PokeAPI endpoint.

This depends on https://github.com/roguedex-dev/roguedex/pull/21 being merged first.

|   Before   |   After   |
|----------|---------|
|    ![Screenshot 2024-05-10 at 4 47 46 PM](https://github.com/roguedex-dev/roguedex/assets/4770500/0269e0ae-9559-4ef2-9ab2-6ef610a9b0bf)    |    ![Screenshot 2024-05-10 at 4 51 25 PM](https://github.com/roguedex-dev/roguedex/assets/4770500/1c5575a0-39ac-4097-bffe-73bc746a3252) |

### How to test
* In both Chrome and Firefox:
    * Load the extension unpacked and see the changes
    * Ensure that the images are loading properly
